.github/workflows/test_workflow.yml

```diff
@@ -24,3 +44,37 @@ jobs:
           submodules: recursive
           token: ${{ secrets.GT_DAXMOBILE }}
           ref: ${{ github.event.inputs.ref }}
+
+      - name: Analyze Maestro Flow Results
```

Comment:
Are these changes to `test_workflow` to stay in here permanently, or changed just for your testing?